growus/growus.py

```diff
@@ -46,5 +51,43 @@ def api_workout_random():
 	response = Response(js, status=200, mimetype="application/json")
 	return response
 
+# inserts a single exercise for a user
+@app.route("/exercises/create", methods = ['POST'])
+def api_exercises_create():
+	exercise_type = request.form['type']
+	user_id = request.form['userId']
+	name = request.form['name']
+	description = request.form['description']
+
+	models.insert_exercise(exercise_type, user_id, name, description)
```

Comment:
When you create a new row in the db, there's some new information, such as `id`. Perhaps you can have `insert_exercise` return the new row? It's possible that `cursor.execute` already does this.